docs/curriculum-topics-resources/open-source-governance.md

```diff
@@ -3,47 +3,45 @@ sidebar_position: 2
+The phrase "open source governance" generally refers to the decision-making model or documented process for how an open source project makes decisions. Sustainable governance is essential if an open source project will last, especially if it involves multiple stakeholders and collaborators. But what governance models or frameworks are sustainable? What makes sense for your open source project? The answer is, it depends.
 
-Documenting and defining a governance model for a project is a radical act of transparency. It is an advance commitment to deciding how decisions are made, who gets to have a say, and how community feedback is factored into the process. Sometimes governance models need to be changed in the future; this is normal. A good governance model captures the core values and beliefs of the project, and gives key stakeholders a voice in the future of the project. It also includes a process to adapt or change, to accommodate unexpected changes in society and norms over time. 
+Documenting and defining a governance model for a project is a radical act of transparency. It is an advance commitment to deciding how decisions are made, who gets to have a say, and how community feedback is factored into the process. Sometimes governance models need to be changed in the future; this is normal. A good governance model captures the core values and beliefs of the project, and gives key stakeholders a voice in the future of the project. It also includes a process to adapt or change, to accommodate unexpected changes in society and norms over time.
 
-In short, governance is the rules or customs by which projects decide who gets to do what or is supposed to do what, how they're supposed to do it, and when. 
+In short, governance is the rules or customs by which projects decide who gets to do what or is supposed to do what, how they're supposed to do it, and when.
 
-This definition of governance can prompt important questions for open source communities seeking to evolve their governance models. Let's explore how. 
+This definition of governance can prompt important questions for open source communities seeking to evolve their governance models. Let's explore how.
 
 Although models and examples are included below, do not take them as an absolute truth. There is room for creativity and adaption; the right governance model for your project might be a combination of the models explained here.
 
 **When you define governance for a project, you need to identify the following five things:**
 _ Josh Berkus, 2020, What is open source project governance?, https://opensource.com/article/20/5/open-source-governance_
-1. What roles can contributors play in the project? 
```

Comment:
I'm also a fan of these questions for determining community governance structures:

https://codebender.org/governance/#21
* Who gets a vote in your group?
* How are new people added to that pool?
* When and how do people lose their votes?
* Where and how do other people find out what has been decided and how it affects them?
* Where and how can non-voters raise issues?

Reply:
Thanks Abby, happy to include this as a resource and additional questions!